src/envoy/http/path_matcher/filter.cc

```diff
@@ -76,7 +76,7 @@ Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
   if (rule == nullptr) {
     rejectRequest(
         Envoy::Http::Code::NotFound,
-        absl::StrCat(method, " ", path,
+        absl::StrCat("`", method, " ", path, "`"
                      " does not match any requirement URI template."));
```

Comment:
How about:
Request with (POST /path) is not defined in the service configuration.

Reply:
How about `` `GET /pets/spot` does not match any defined routes`` ? 

Currently, the service configuration is transparent to downstream. It is unnecessary to mention this.